### ISSUE TITLE
golangci-lint run error due to depguard

### ISSUE THREAD

**Issue by psergee** [golangci/golangci-lint#3876]
### Welcome

- [X] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, etc.).
- [X] Yes, I've tried with the standalone linter if available (e.g., gocritic, go vet, etc.). (https://golangci-lint.run/usage/linters/)

### Description of the problem

golangci-lint fails with error from depguard.
How to reproduce:
Install 1.53.0 version.
```
$ golangci-lint --version
ERRO [linter] depguard: create analyzer: couldn't expand $gostd: could not read GOROOT directory: open src: no such file or directory
must have an Allow and/or Deny package list 
$ golangci-lint --help
ERRO [linter] depguard: create analyzer: couldn't expand $gostd: could not read GOROOT directory: open src: no such file or directory
must have an Allow and/or Deny package list
```
Expected: no errors from linters are expected if I run commands which are not related to linters: --version or --help.
`depguard` linter is not enabled in config file.

### Version of golangci-lint

<details>

```console
$ golangci-lint --version
ERRO [linter] depguard: create analyzer: couldn't expand $gostd: could not read GOROOT directory: open src: no such file or directory
must have an Allow and/or Deny package list
```
The command fails and does not show the version.
1.53.0 is installed.

### Configuration file

<details>

```console
$ cat .golangci.yml
linters:
  disable-all: true
  enable:
    - lll
    - govet
    - gofmt
    - stylecheck

linters-settings:
  lll:
    line-length: 100
    tab-width: 4
  gofmt:
    simplify: false
  stylecheck:
    checks: ["all"]
```

</details>

### Go environment

<details>

```console
$ go version && go env
go version go1.19.4 linux/amd64
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/psergee/.cache/go-build"
GOENV="/home/psergee/.config/go/env"
GOEXE=""
GOEXPERIMENT=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOINSECURE=""
GOMODCACHE="/home/psergee/go/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/home/psergee/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/local/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GOVCS=""
GOVERSION="go1.19.4"
GCCGO="gccgo"
GOAMD64="v1"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="/home/psergee/apps/go.mod"
GOWORK=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -Wl,--no-gc-sections -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build3027707310=/tmp/go-build -gno-record-gcc-switches"
```

</details>

### Verbose output of running

<details>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
ERRO [linter] depguard: create analyzer: couldn't expand $gostd: could not read GOROOT directory: open src: no such file or directory
must have an Allow and/or Deny package list
```

</details>

### Code example or link to a public repository

<details>

```go
// add your code here
```

</details>

**Comment by ldez** [comment]
duplicate of #3862, #3870, #3871,https://github.com/golangci/golangci-lint-action/issues/760, https://github.com/golangci/golangci-lint-action/issues/761, https://github.com/golangci/golangci-lint-action/issues/759
